machAeroTutorials/aero_gridRefinementStudy.rst

```diff
@@ -66,8 +66,114 @@ An example of grid convergence plot for a family of RAE 2822 Airfoil meshes is i
 
     Figure 1: Grid convergence plot for RAE 2822 Transonic Airfoil.
 
+
+
+2. Grid Refinement Study on Airfoils
+======================================
+
+Instead of using the :code:`cgns_utils coarsen` feature, we can easily make the finer or coarsen meshes with the help of  :code:`prefoil` package.
```

Comment:
Instead of using `code`, you should just use double backticks, e.g. ` ``prefoil`` `.